liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -68,7 +63,13 @@ protected void installApp(Artifact artifact) throws Exception {
             throw new MojoExecutionException(messages.getString("error.install.app.missing"));
         }
         
-        File destDir = new File(serverDirectory, appsDirectory);
+        setAppsDirectory();
```

Comment:
The previous default 'dropins' is removed. Thus, this method will return 'dropins'(the previous default) for other goals(i.e. called within exportParametersToXml() method in PluginConfigSupport.java) except install-apps. Within the 'install-apps' goal, 'setAppsDirectory()' will override the default value by the smart default.